package.json

```diff
@@ -112,6 +112,7 @@
         "cesium-dnd": "1.1.0",
         "core-js": "3.26.1",
         "crypto-js": "4.1.1",
+        "csv-parse": "^5.3.3",
```

Comment:
Please remember to remove the `^` from newly added deps so we can all have the same local environment and it'll be the same as testing environment always!

Reply:
Oh... I'm sorry, I forgot it. I fixed it.
[chore: remove ^ versioning](https://github.com/reearth/reearth-web/pull/382/commits/854e6a1ae664324cda3b7be59e2340e3a4daac37)